flaky/_flaky_plugin.py

```diff
@@ -349,3 +407,35 @@ def _get_test_method_and_name(cls, test):
             `tuple` of `callable`, `unicode`
         """
         raise NotImplementedError
+
+    @classmethod
+    def _make_test_method_flaky(cls, test, max_runs=None, min_passes=None):
+        """
+        Make a given test method flaky.
+        :param test:
+            The test in question.
+        :type test:
+            :class:`Function`
+        :param max_runs:
+            The value of the FlakyNames.MAX_RUNS attribute to use.
+        :type max_runs:
+            `int`
+        :param min_passes:
+            The value of the FlakyNames.MIN_PASSES attribute to use.
+        :type min_passes:
+            `int`
+        """
+        if max_runs is None:
+            max_runs = 2
+        if min_passes is None:
+            min_passes = 1
```

Comment:
Why not just use default values of 2 and 1 for the ``max_runs`` / ``min_passes`` params?